src/monitor-api-requests.ts

```diff
@@ -0,0 +1,158 @@
+import { MonitorParams } from "monitors";
+import { Response } from "node-fetch";
+import * as Serverless from "serverless";
+import fetch from "node-fetch";
+
+export class InvalidAuthenticationError extends Error {
+  constructor(message: string) {
+    super(...message);
+    this.name = "Invalid Authentication Error";
+    this.message = message;
+  }
+}
+
+interface QueriedMonitor {
+  query: string;
+  id: number;
+  name: string;
+  tags: string[];
+}
+
+export async function createMonitor(
+  serverlessMonitorId: string,
+  monitorParams: MonitorParams,
+  monitorsApiKey: string,
```

Comment:
Instead of passing the `monitorsApiKey` and `monitorsAppKey` into every API call, I think it might be cleaner if we just pulled these values out of the config object within the function. The API key and app key are going to be the same for all API calls each time we deploy, so I don't think we need to re-specify them for every API call.

That way we can reduce the number of arguments being passed to each function! This will help make them more readable.